lambdas/build-client-oauth-response/src/main/java/uk/gov/di/ipv/core/buildclientoauthresponse/BuildClientOauthResponseHandler.java

```diff
@@ -184,14 +184,15 @@ public Map<String, Object> handleRequest(JourneyRequest input, Context context)
 
             var isReproveIdentity = clientOAuthSessionItem.getReproveIdentity();
             if (Boolean.TRUE.equals(isReproveIdentity)) {
+                Vot vot = ipvSessionItem.getVot();
                 auditService.sendAuditEvent(
                         AuditEvent.createWithoutDeviceInformation(
                                 AuditEventTypes.IPV_ACCOUNT_INTERVENTION_END,
                                 configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
                                 auditEventUser,
                                 AuditExtensionAccountIntervention.newReproveIdentity(
                                         ipvSessionItem.getErrorCode() == null
-                                                && Vot.P2.equals(ipvSessionItem.getVot()))));
+                                                && (Vot.P2.equals(vot) || Vot.P1.equals(vot)))));
```

Comment:
I think this might want to be checking if the vot (attained) is in the vtr. Tbf I'm a bit uncertain of what "success" means on the extension

Reply:
Yes a successful reprove would mean the attained VOT meets the requested VTR